build_debian.sh

```diff
@@ -177,6 +177,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT useradd -G sudo,docker $USERNAME -c "$DEFAUL
 ## Create password for the default user
 echo "$USERNAME:$PASSWORD" | sudo LANG=C chroot $FILESYSTEM_ROOT chpasswd
 
+if [ "$SONIC_ROUTING_STACK" == "frr" ]; then
+    sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -g $FRR_USER_GID frr
+    sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -g $FRR_VTY_GID frrvty
+    sudo LANG=C chroot $FILESYSTEM_ROOT useradd -u $FRR_USER_UID -g $FRR_USER_GID -M -s /bin/false frr
```

Comment:
>useradd -u $FRR_USER_UID -g $FRR_USER_GID -M -s /bin/false frr [](start = 40, length = 62)

Fixing UID or GID numbers in host or in docker container is not good. What we can safely assume is that the root user id and root group id on both side are 0.

Reply:
Thank you for the feedback. FRR is expecting frr uid/gid. At this point in time we also don't want frr user to have root user/group privileges.